### PR TITLE
{system}[dummy] CUDA 9.0.176

### PR DESCRIPTION
**Pull request by smoors**
{system}[dummy] CUDA 9.0.176

### CODE REVIEW

File: easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://developer.nvidia.com/compute/cuda/%(version_major_minor)s/Prod/local_installers/']
+sources = ['cuda_%(version)s_384.81_linux-run']
+checksums = ['96863423feaa50b5c1c5e1b9ec537ef7ba77576a3986652351ae43e66bcd080c']
+
+modextravars = {'CUDA_HOME': '%(installdir)s'}
```
Comment (boegel) [inline easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb]: @smoors This is no longer needed, it's handled by the `EB_CUDA` easyblock since EasyBuild 2.5.0

Comment (smoors) [reply easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb]: @boegel you mean I can remove the last line, right?

Comment (smoors) [reply easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb]: after adding the EB_CUDA easyblock

Comment (boegel) [reply easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb]: No need to specify it to use `EB_CUDA`, it already does (if no `easyblock =` is specified, EasyBuild will use the software-specific easyblock based on `name`).

Just drop the `modextravars` line (and the empty line below it).

File: easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+easyblock = 'EB_CUDA'
```
Comment (boegel) [inline easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb]: @smoors As I mentioned, this is not needed, EasyBuild will use `EB_CUDA` by default based on the software name if no `easyblock` is specified...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2690.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f54e2ff174aef766c4cb846c5816cb0d for a full test report.

File: easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb (hunk 2 above)
Comment (smoors) [reply easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb]: oops sorry, I thought I deleted it..

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2026.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d84bae1ef931afb7fea13f547c8caa42 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**